jmclient/jmclient/configure.py

```diff
@@ -262,6 +262,11 @@ def jm_single():
 #        2. You cannot change the type of a pre-existing wallet.
 native = true
 
+# Display zpub for native segwit wallet, and ypub for non-native segwit wallet.
+# If set to false, the extended public key will always display in the xpub
+# format.
+display_ypub_zpub = true
```

Comment:
Maybe mention that this affects only Qt GUI, not `wallet-tool.py`? Also probably could be renamed to `gui_display_ypub_zpub` or `qt_display_ypub_zpub`.

Reply:
Yeah, I had been thinking whether or not we want this option to be gui only, or should we convert xpub to ypub/zpub for the console as well.

I guess it makes sense to rename this settings for now as it's only affecting the gui today.

Reply:
Done